common/character_templates/dvg_agitators.txt

```diff
@@ -38,3 +38,21 @@
 		chance = 100
 	}
 }
+
+dvg_giancarlo_iannarella_template = {
+	first_name = Giancarlo
+	last_name = Iannarella
+	is_agitator = yes
+	female = no
+	dna = dna_Iannarella
+	age = 31
+	culture = cu:venetian
+	interest_group = ig_intelligentsia
+	ideology = ideology_positivist

```

Comment:
Failed to find a valid event target link 'ideology_positivist' at  file: events/dvg_venice.txt line: 1959